tex/gregoriotex-main.tex

```diff
@@ -214,8 +214,7 @@
 \let\gre@pitch@underbrace\gre@pitch@b %
 \let\gre@pitch@overbraceglyph\gre@pitch@g %
 \let\gre@pitch@bar\gre@pitch@g %
-\let\gre@pitch@dummy\gre@pitch@g %
-\let\gre@pitch@nominal\gre@pitch@e %
+\let\gre@pitch@dummy\gre@pitch@e % not above or below staff, even if 2 lines
```

Comment:
This seems to be used a lot, and I didn't check that e would be okay in all cases. It doesn't break any tests.